### PR TITLE
New/pull request view

### PR DESCRIPTION
**Pull request by TheSubliminal**

### CODE REVIEW

File: server/data/models/pullReviewer.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = (sequelize) => {
+  const pullReviewers = sequelize.define('pullReviewers', {}, {
+    /* timestamps: true,
+    paranoid: true, */
```
Comment (denterekhov) [inline server/data/models/pullReviewer.js]: Is it necessary for some reason?

Comment (TheSubliminal) [reply server/data/models/pullReviewer.js]: Yes, it is, I forgot to uncomment

File: server/data/db/associations.js [hunk 2]
```diff
@@ -88,6 +90,9 @@ module.exports = (models) => {
   PullRequest.belongsTo(Branch, { as: 'toBranch', foreignKey: 'toBranchId' });
   PullComment.belongsTo(User);
   PullComment.belongsTo(PullRequest, { as: 'pull' });
+  PullReviewer.belongsTo(User);
+  PullReviewer.belongsTo(PullRequest, { as: 'pull' });
+  PullReviewer.belongsTo(ReviewStatus, { as: 'status' });
```
Comment (denterekhov) [inline server/data/db/associations.js]: Should we need to add User.hasMany(PullReviewer) and so on?

Comment (TheSubliminal) [reply server/data/db/associations.js]: In this case hasMany association will add userId to PullReviewer, but it is already done by belongsTo association, so I don't think we need to do this.

Comment (denterekhov) [reply server/data/db/associations.js]: Got it, thanks.

**Pull request merged**